CHANGELOG.md

```diff
@@ -16,8 +16,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Delete legacy chart, used by draughtsman.
 
-[0.2.0]: https://github.com/giantswarm/dex-app/tag/v0.2.0
```

Comment:
When we are cleaning things here we should also have `[Unreleased]` headers and `/compare/` links at the bottom.